WordPress/Classes/ViewRelated/Support/SupportTableViewController.swift

```diff
@@ -26,6 +32,11 @@ class SupportTableViewController: UITableViewController {
 
     required convenience init() {
         self.init(style: .grouped)
+        NotificationCenter.default.addObserver(self, selector: #selector(showNotificationIndicator(_:)), name: .ZendeskPushNotificationReceivedNotification, object: nil)
```

Comment:
Since we're calling the overridden `init(style:)` initializer, it might be better to add the observer in that method vs here.  That way it will always be set up regardless of which intializer is called.  (Also it would make `init(style:)` more than just a pass-through.)